File: 2026-08/src/day_26/prms_181900/Solution.cpp
#include <bits/stdc++.h>
using namespace std;

string solution(string my_string, vector<int> indices) {
    for (int i : indices) {
        my_string[i] = '\0';
    }
    erase(my_string, '\0');

    return my_string;
}

File: 2026-08/src/day_26/prms_181900/Solution.java
package day_26.prms_181900;

class Solution {
    public String solution(String my_string, int[] indices) {
        char[] arr = my_string.toCharArray();
        for (int i : indices) {
            arr[i] = '\u0000';
        }

        StringBuilder sb = new StringBuilder();
        for (char c : arr) {
            if (c != '\u0000') sb.append(c);
        }

        return sb.toString();
    }
}
